pkg/rpc/core/utils_test.go

```
@@ -37,12 +38,9 @@ func TestTruncateNodeID(t *testing.T) {
 	// Test case 3: Invalid hex string
 	t.Run("invalid hex string", func(t *testing.T) {
 		invalidHexStr := "not-a-hex-string-at-all-and-should-be-long-enough-to-pass-length-check"
-		// Ensure it would be long enough if it were hex
-		if len(invalidHexStr) < NodeIDByteLength*2 {
-			invalidHexStr = strings.Repeat("x", NodeIDByteLength*2) // Make it long enough but still invalid hex
-		}
+		require.GreaterOrEqual(t, len(invalidHexStr), NodeIDByteLength*2)
```

more readable and fails fast when const is changed